server/test/integration/post.spec.js

```diff
@@ -106,4 +106,81 @@ describe('Posts API', () => {
     expect(response2.status).toBe(404);
     done();
   });
+
+  it('tests the delete post method', async done => {
+    const postData = {
+      title: 'Test post',
+      body: 'This is the body for a test post',
+    };
+
+    const response = await supertest(app)
+      .post('/posts')
+      .send(postData);
+
+    const createdPost = response.body;
+    const url = '/posts/';
+
+    const response1 = await supertest(app).delete(url.concat(createdPost._id));
+
+    expect(response1.status).toBe(200);
+    done();
+  });
+
+  it('tests the deletion of already deleted post and return error message', async done => {
+    const postData = {
+      title: 'Test post',
+      body: 'This is the body for a test post',
+    };
+
+    const response = await supertest(app)
+      .post('/posts')
+      .send(postData);
+
+    expect(response.status).toBe(201);
+
+    const createdPost = response.body;
+    const url = '/posts/';
+
+    const response1 = await supertest(app).delete(url.concat(createdPost._id));
+
+    expect(response1.status).toBe(200);
+
+    const response3 = await supertest(app).get(url);
```

Comment:
Is this required by the test? The response3 variable is not used anywhere apart from the console log.

Reply:
I will change that! Good spotting, it was used for me to see that the deletion actually went through.